services/src/main/scala/cromwell/services/keyvalue/impl/KeyValueDatabaseAccess.scala

```diff
@@ -1,34 +1,7 @@
 package cromwell.services.keyvalue.impl
```

Comment:
Since you're making hefty changes to the KV, can you go ahead and rename it to something more apropos as per the discussion last week?